hi-ml-azure/src/health_azure/utils.py

```diff
@@ -673,6 +673,26 @@ def download_checkpoint_if_necessary(self) -> Path:
         return self.local_checkpoint_path
 
 
+def get_checkpoint_downloader(ckpt_run_id: str, outputs_folder: Path) -> CheckpointDownloader:
+    """Creates an instance of CheckpointDownloader for the specified run_id and downloads the corresponding checkpoint.
+
+    :param ckpt_run_id: The Azure ML run_id of the run from which to load the checkpoint.
+    :param outputs_folder: The outputs folder where to download the checkpoint.
+    :return: A downloader instance for the specified run_id.
+    """
+    from health_ml.utils.common_utils import CHECKPOINT_FOLDER, DEFAULT_AML_UPLOAD_DIR
+    from health_ml.utils.checkpoint_utils import LAST_CHECKPOINT_FILE_NAME_WITH_SUFFIX
+    downloader = CheckpointDownloader(
```

Comment:
That was used in cpath to load ssl checkpoints, I didn't want to duplicate it so I moved it to health_azure, looking at you comment below, It might not be the best solution.